README.md

```diff
@@ -22,7 +22,40 @@ Or install it yourself as:
 
 ## Usage
 
-To create a `SageoneSdk::Client`, you need to provide your access_token and signing_secret:
+# Authorization
```

Comment:
If these headings should be nested under `## Usage`, they need to be h3's : `### Authorization`